### PR TITLE
Add bootstrap

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by dougalsutherland** [comment]
See discussion at https://github.com/conda-forge/staged-recipes/pull/4044#issuecomment-332019807.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge.

**Comment by isuruf** [comment]
```
INFO conda_build.build:bundle_conda(863): Packaging bootstrap-3.3.7-py_0
number of files: 0
```

File: recipes/bootstrap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  noarch: True
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    
+test:
+  imports:
```
Comment (scopatz) [inline recipes/bootstrap/meta.yaml]: No tests

File: recipes/bootstrap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/twbs/bootstrap/archive/v3.3.7.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: True
```
Comment (dougalsutherland) [inline recipes/bootstrap/meta.yaml]: Should be `noarch: generic` I think; see [docs](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#architecture-independent-packages).

File: recipes/bootstrap/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{% set name = "bootstrap" %}
+{% set version = "3.3.7" %}
+{% set sha256 = "859db5ef33728c6bfbea0ae8891c5e928176b9a69478761fe8363e5e8d354945" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/twbs/bootstrap/archive/v3.3.7.tar.gz
```
Comment (dougalsutherland) [inline recipes/bootstrap/meta.yaml]: `v{{ version }}.tar.gz`

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge.

**Comment by scopatz** [comment]
Mergign since no-arch and travis will fail anyway/.

**Comment by sannykr** [comment]
@scopatz : Thanks.

**Pull request merged**